tools/chibi-genstatic

```diff
@@ -425,8 +425,7 @@
           (set! *features* features))
       (let* ((c-libs+c-files (find-c-libs includes excludes cfiles))
              (c-libs (car c-libs+c-files))
-             (c-files (cdr c-libs+c-files))
-             (inline? #t))
```

Comment:
I can't remember why I did this, but we shouldn't change the default.  We should make inline? default to true above and provide a --no-inline option or similar.